lib/Common/Memory/LargeHeapBlock.cpp

```diff
@@ -197,15 +200,18 @@ LargeHeapBlock::~LargeHeapBlock()
 #ifdef RECYCLER_PAGE_HEAP
     if (this->pageHeapAllocStack != nullptr)
     {
-        this->pageHeapAllocStack->Delete(&NoCheckHeapAllocator::Instance);
+        if (this->pageHeapAllocStack != (StackBackTrace*)1)
```

Comment:
:) OOM when allocating allocStack, I set the pointer to non-null to indicated we did try to allocate for allocStack. when allocating for freeStack, we have check if allocStack is non-null, that check will fail if OOM while allocating allocStack

---
In reply to: [60992519](https://github.com/Microsoft/ChakraCore/pull/711#discussion_r60992519) [](ancestors = 60992519)